x-pack/plugins/translations/translations/fr-FR.json

```diff
@@ -31216,6 +31216,9 @@
     "xpack.triggersActionsUI.timeUnits.hourLabel": "{timeValue, plural, one {heure} other {heures}}",
     "xpack.triggersActionsUI.timeUnits.minuteLabel": "{timeValue, plural, one {minute} other {minutes}}",
     "xpack.triggersActionsUI.timeUnits.secondLabel": "{timeValue, plural, one {seconde} other {secondes}}",
+    "xpack.triggersActionsUI.toolbar.bulkActions.clearSelectionTitle": "Effacer la sélection",
```

Comment:
You do not need to add the new translation because at each release, we have a process to rebuild all these translations. It is ok to remove it. You get error when the key duplicate.